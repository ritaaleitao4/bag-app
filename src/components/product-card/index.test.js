import ProductCard from '.';

const mockItem = {
    id: 'item1',
    name: 'BRAND 1',
    image: 'https://via.placeholder.com/130x175',
    price: '100.40',
};

describe('product card component', () => {
    it('it renders correctly', () => {
        expect(ProductCard(mockItem)).toMatchSnapshot();
    });
});
