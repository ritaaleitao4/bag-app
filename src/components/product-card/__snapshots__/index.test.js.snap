// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`product card component it renders correctly 1`] = `

<div class="product-card">
  <div class="remove-btn">
    <img src="test-file-stub">
  </div>
  <div class="image-container">
    <img class="image"
         src="https://via.placeholder.com/130x175"
    >
  </div>
  <div class="info-container alignCenter">
    <div class="brandName">
      BRAND 1
    </div>
    <div class="price">
      â‚¬100.40
    </div>
  </div>
</div>

`;
